app/controllers/pet_applications_controller.rb

```diff
@@ -0,0 +1,16 @@
+class PetApplicationsController < ApplicationController
```

Comment:
Making a joins controller is a great idea, but you don't necessarily need a `index` or `show` in it. This joins controller can be helpful for implementing the functionality of adding a pet to an application. When a user clicks the "Add Pet" button, it could route to a `create` action in this joins controller because what that button is triggering is the **creation** of a joins record! So, that's a typical use case for a joins controller